internal/d2mod/randomizer/randomizer.go

```diff
@@ -158,7 +148,7 @@ func getProps(p propGetter) (Props, Items) {
 			if err == nil {
 				lvl = mbLvl
 			}
-		} else if p.fileName == runesTxt.FileName { //Runes.txt doesn't have a level column
+		} else if p.fileName == runes.FileName { //Runes.txt doesn't have a level column
```

Comment:
i think this comment is out of date

Reply:
no, this is the else case from "if p.levelOffset >= 0" the runeword function sets the level offset to -1 because there isn't a qlvl column, and we have to do the thing where we parse the runes and get the level of the highest one. I could probably expand this comment if to describe better what it is pointing out.